### ISSUE TITLE
Audit redesign for accessibility

### ISSUE THREAD

**Issue by fg-j** [paketo-buildpacks/paketo-website#159]
Once we're happy with the visuals of the homepage on mobile, make sure it's meeting accessibility standards. 

Collect resources for assessing accessibility below

*A/C*
Resolve all issues on WCAG guidelines [checklist](https://www.a11yproject.com/checklist).

## Resources
- https://www.a11yproject.com/checklist/ A checklist that distills down the WCAG guidelines
- https://www.webaccessibility.com/ a tool that will scan a URL and give an accessibility score and point out violations

Document any key learnings in the comments.

**Comment by fg-j** [comment]
Currently, we have no way to pause the keyshape animation in the How it Works section. For now, hiding the animation when the user `prefers-reduced-motion`. Ideally, we can still display some sort of image or set the animation not to autoplay. May create a follow-on issue for this.

**Comment by fg-j** [comment]
I ran through the checklist provided by A11y project. Below are elements of the checklist that haven't yet been directly fixed/addressed:

- [ ] Make sure there is a visible focus style for interactive elements that are navigated to via keyboard input.
- [ ] Remove invisible focusable elements.
- [ ] Provide a skip link and make sure that it is visible when focused.
- [ ] Identify links that open in a new tab or window.
- [ ] Make sure that media does not autoplay.
- [ ] Ensure that media controls use appropriate markup.
- [ ] Check to see that all media can be paused.

**Comment by fg-j** [comment]
Closing for now. We've made significant improvements to accessibility as compared to the original site. Further issues can be opened to address any outstanding critical problems.
